data/remotes.d/fwupd.conf

```diff
@@ -0,0 +1,6 @@
+[fwupd Remote]
+
+# this remote provides metadata shipped with the fwupd package
+Enabled=true
+Type=local
+File=@datadir@/fwupd/remotes.d/fwupd/metadata.xml
```

Comment:
I think it's fine uncompressed for now; I'll ask the meson guys the best way to do this.